pkg/api/postage.go

```diff
@@ -233,15 +233,15 @@ func (s *Service) postageGetAllStampsHandler(w http.ResponseWriter, _ *http.Requ
 		}
 
 		batches = append(batches, postageBatchResponse{
-			BatchID:       b.ID,
-			Value:         bigint.Wrap(b.Value),
-			Start:         b.Start,
-			Owner:         b.Owner,
-			Depth:         b.Depth,
-			BucketDepth:   b.BucketDepth,
-			Immutable:     b.Immutable,
-			StorageRadius: b.StorageRadius,
-			BatchTTL:      batchTTL,
+			BatchID:     b.ID,
+			Value:       bigint.Wrap(b.Value),
+			Start:       b.Start,
+			Owner:       b.Owner,
+			Depth:       b.Depth,
+			BucketDepth: b.BucketDepth,
+			Immutable:   b.Immutable,
+			// StorageRadius: b.StorageRadius,
```

Comment:
@istae Should we remove this field from the API response? I think this is no longer tracked in the batchstore right?

Reply:
yes, I believe so, we use a common storage radius everywhere now